bin/prettier.js

```diff
@@ -16,8 +16,8 @@ const args = [
   "src/*.js",
   "src/*/*.js",
   "src/components/**/*.css",
-  "src/*/!(mochitest)**/*.js",
-  "src/*/!(mochitest)*/**/*.js"
+  // "src/*/!(mochitest)**/*.js",
```

Comment:
When I comment out this lines prettier script runs on windows. It seems to me that the CLI wants to jump into the globbing which can not be done on Windows